Makefile

```diff
@@ -225,7 +228,7 @@ download: $(BUILDDIR)/$(APP).elf
 	$(INSTALLDIR)/bin/patserdow -v $(COM_PORT) $<
 
 fpgaexec: $(BUILDDIR)/$(APP).elf
-	$(INSTALLDIR)/bin/patserdow $(COM_PORT) $<
+	$(INSTALLDIR)/bin/patserdow $(COM_POpthread_mutex_testRT) $<
```

Comment:
What does COM_POpthread_mutex_testRT mean?

Reply:
Whoops this is a mistake... from testing i must have copied "pthread_mutex_test" in

Reply:
I have reverted the mistake an pushed it to master